polyfill/intersection-observer-test.js

```diff
@@ -969,6 +970,12 @@ describe('IntersectionObserver', function() {
         iframeDoc.write('</style>');
         iframeDoc.close();
 
+        // Ensure the documentElement and body are always sorted on top.
```

Comment:
Not sure I follow this comment. Can you explain what this is ensuring?

Reply:
I was just trying to follow `sortRecords` function that was available in these tests. It's used to make records order predictable, which totally makes sense. So I wanted to make sure that if `<html>` or `<body>` are observed, they'd always be sorted in the predictable order as well, along with all the other targets.

Reply:
I added the comment in the code about `sortRecords`.